includes/AddOns/TypesForSteelDesign/BoundaryCondition.js

```diff
@@ -0,0 +1,546 @@
+ * @param {Number} eccentricity_z           Eccentricity in z'-axis, can be undefined (is not set, 0.0 as default)
+ */
+SteelDesignBoundaryCondition.prototype.SetAdditionalParametersForEnd = function (rotation,
+    eccentricity_type_z,
+    eccentricity_x,
+    eccentricity_z) {
+    setSteelDesignNodalSupportsAdditionalParameters(this.boundary_condition, this.boundary_condition.nodal_supports.row_count(), rotation, eccentricity_type_z, eccentricity_x, eccentricity_z);
+};
+
+/**
+ * Sets Additional parameters for intermediate node
+ * @param {Number} row                      Row, first intermediate row begins on index 2
+ * @param {Number} rotation                 Rotation, can be undefined (is not set, 0.00 as default)
+ * @param {String} eccentricity_type_z      Type of eccentricity in z-axis (NONE, AT_UPPER_FLANGE, AT_LOWER_FLANGE, USER_VALUE), can be undefined (is not set, USER_VALUE as default)
+ * @param {Number} eccentricity_x           Eccentricity in x'-axis, can be undefined (is not set, 0.0 as default)
+ * @param {Number} eccentricity_z           Eccentricity in z'-axis, can be undefined (is not set, 0.0 as default)
+ */
+SteelDesignBoundaryCondition.prototype.SetAdditionalParametersForIntermediateRow = function (row,
+    rotation,
+    eccentricity_type_z,
+    eccentricity_x,
+    eccentricity_z) {
+    setSteelDesignNodalSupportsAdditionalParameters(this.boundary_condition, row, rotation, eccentricity_type_z, eccentricity_x, eccentricity_z);
+};
+
+/**
+ * Sets Intermediate nodes state
+ * @param {Boolean} intermediate_nodes  Intermediate nodes, can be undefined (true as default)
+ */
+SteelDesignBoundaryCondition.prototype.IntermediateNodes = function (intermediate_nodes) {
+    if (typeof intermediate_nodes === "undefined") {
+        intermediate_nodes = true;
+    }
+    this.boundary_condition.intermediate_nodes = intermediate_nodes;
+};
+
+/**
+ * Sets Different properties state for nodal supports
+ * @param {Boolean} different_properties_supports   Different properties, can be undefined (true as default)
+ */
+SteelDesignBoundaryCondition.prototype.DifferentPropertiesForNodalSupports = function (different_properties_supports) {
+    if (typeof different_properties_supports === "undefined") {
+        different_properties_supports = true;
+    }
+    this.boundary_condition.different_properties_supports = different_properties_supports;
+};
+
+/**
+ * Sets Hinge releases for start node sequence
+ * @param {Number} release_in_y         Release in y, can be undefined (is not set, false as default)
+ * @param {Number} release_about_x      Release about x, can be undefined (is not set, false as default)
+ * @param {Number} release_about_z      Release about z, can be undefined (is not set, false as default)
+ * @param {Number} release_warping      Release of warping, can be undefined (is not set, false as default)
+ */
+SteelDesignBoundaryCondition.prototype.MemberHingesForStart = function (release_in_y,
```

Comment:
```suggestion
SteelDesignBoundaryCondition.prototype.Set
```suggestion
SteelDesignBoundaryCondition.prototype.MemberHingesForStart = function (release_in_y,
```MemberHingesForStart = function (release_in_y,
```